libs/scripts/scripts.js

```diff
@@ -38,6 +38,36 @@ export async function loadLCP(blocks) {
   }
 }
 
+export async function fetchLib(type, url = '') {
+  const namespace = `${type}Library`;
+  window.milo = window.milo || {};
+  if (!type || window.milo[namespace]) return;
+  const endpoint = (url !== '') ? `${url}/docs/${type}-library.json` : `${window.location.origin}/docs/${type}-library.json`;
+  const resp = await fetch(endpoint);
+  if (resp.ok) {
+    try {
+      const json = await resp.json();
+      window.milo[namespace] = {};
+      json.data.forEach((item) => {
+        const itemValues = {};
+        // eslint-disable-next-line no-restricted-syntax
+        for (const [key, value] of Object.entries(item)) {
+          if (key !== 'key') {
+            itemValues[key] = value;
+          }
+        }
+        window.milo[namespace][item.key] = itemValues;
+      });
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.log(`Could not make ${type} library`);
+    }
+  } else {
+    // eslint-disable-next-line no-console
+    console.log(`Could not get ${type} library`);
+  }
+}
+
```

Comment:
This whole function looks like it needs a refactor. Some good ideas, but far too many eslint-disables.